### PR TITLE
Binding failure story

### PR DESCRIPTION
**Pull request by ssaucedo**

### CODE REVIEW

**Comment by kshakir** [comment]
@geoffjentry @jmthibault79 @dvoet I'm googling also, but any inspirational ways of implementing a basic test for this commit?

File: src/main/scala/org/broadinstitute/dsde/vault/common/util/ServerInitializer.scala [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import akka.pattern.ask
+import akka.util.Timeout
+import spray.can.Http
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+object ServerInitializer {
+
+  def startWebServiceActors(props: Props, webserviceInterface: String, webservicePort: Int, timeoutLong: Long, actorSystem: ActorSystem): Unit = {
```
Comment (kshakir) [inline src/main/scala/org/broadinstitute/dsde/vault/common/util/ServerInitializer.scala]: Overall looks good. Minor preferences:
- Don't be like me. Add scaladoc.
- Rename the `timeoutLong` to `timeoutSeconds: Long` (better name for consumers), or better yet, maybe take the value in as a `timeoutDuration: scala.concurrent.duration.Duration`.

File: src/main/scala/org/broadinstitute/dsde/vault/common/util/ServerInitializer.scala [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package org.broadinstitute.dsde.vault.common.util
+
+import akka.actor.Status.{Failure}
```
Comment (kshakir) [inline src/main/scala/org/broadinstitute/dsde/vault/common/util/ServerInitializer.scala]: Remove `{}`.

**Comment by kshakir** [comment]
Minor updates, plus (possible?) tests, and good to go!

**Comment by geoffjentry** [comment]
@kshakir Personally I'd file this under the "not worth testing" category but I've been told that my version of that category is far too broad so you probably shouldn't listen to me :)

**Comment by davidangb** [comment]
FYI, I gave Santiago et al guidance that an automated test is not worth the cost:benefit - Walter has tested this manually. (but if you can think of an easy automated test, great!)

**Comment by kshakir** [comment]
:+1: on skipping tests for this.

FYI if someone is interested in the future-- here's what my searches turned up when I went to look at/near existing tests for `spray.can.Http` and `akka.io.IO`.

https://github.com/spray/spray/tree/c76b75370f231f151df2283d300a9bcfdc6f2a9c/spray-can-tests/src/test/scala/spray/can/client

https://github.com/akka/akka/tree/67a9e622543b00582ad55cd76a2a2871741c32c8/akka-actor-tests/src/test/scala/akka/io

**Comment by kshakir** [comment]
:+1: Please squash into a single commit and then merge to develop. After jenkins builds, the new version will be in https://artifactory.broadinstitute.org/artifactory. Get the new version ID from there, and use it in DM/API.

For DM and API, please ping someone else in HipChat to review those PRs.

Thanks!

**Pull request merged**